core/db/migrate/20160318130914_migrate_default_to_current_prices.rb

```diff
@@ -0,0 +1,34 @@
+class MigrateDefaultToCurrentPrices < ActiveRecord::Migration
+  def up
+    say_with_time 'Setting default prices to be valid from now' do
```

Comment:
I don't think the `say_with_time` is necessary when all it's wrapping is an `execute`. But it doesn't really hurt to have it.